src/components/ContentWrapper.jsx

```diff
@@ -48,33 +47,33 @@ const ContentWrapper = (props) => {
   const [collapsed, setCollapsed] = useState(false);
 
   const { routeExperimentId, experimentData, children } = props;
-  const { navigateTo } = useAppRouter();
-  const router = useRouter();
-  const route = router?.route || '';
+  const { navigateTo, currentModule } = useAppRouter();
 
   const currentExperimentIdRef = useRef(routeExperimentId);
   const activeProjectUuid = useSelector((state) => state?.projects?.meta?.activeProjectUuid);
   const activeProjectExperimentID = useSelector((state) => (
     state?.projects[activeProjectUuid]?.experiments[0]));
 
   const activeProject = useSelector((state) => state.projects[activeProjectUuid]);
-
   const samples = useSelector((state) => state.samples);
 
+  // This use effect block is used to get the active experiment ID in cases where experimentId is not evident in the route
+  // This happens in data-management, where we set the active experimentId to be the experimentId of the project
+  // ExperimentId in the route is used as a higher priority
   useEffect(() => {
     if (!activeProjectExperimentID && !routeExperimentId) return;
```

Comment:
if this useEffect is used only for data management module, you can also replace this if statement with `if(currentModule !== modules.DATA_PROCESSING)`

Reply:
Hmm... you're right. I'll try it out.

Reply:
This is working, I've implemented the changes.